tools/convert_feat_to_wholegraph.py

```diff
@@ -16,160 +16,255 @@
 """
 import argparse
+import gc
 import json
+import logging
 import os
 import re
-import torch
+
 import dgl
-import logging
 import pylibwholegraph.torch as wgth
+import torch
 
 
-def get_node_feat_info(folder, node_feat_names):
-    # per node type feature
+def get_node_feat_info(node_feat_names):
+    """Process node feature names"""
     fname_dict = {}
     for feat_name in node_feat_names:
         feat_info = feat_name.split(":")
-        assert len(feat_info) == 2, \
-                f"Unknown format of the feature name: {feat_name}, " + \
-                "must be NODE_TYPE:FEAT_NAME"
+        assert len(feat_info) == 2, (
+            f"Unknown format of the feature name: {feat_name}, "
+            + "must be NODE_TYPE:FEAT_NAME"
+        )
         ntype = feat_info[0]
-        assert ntype not in fname_dict, \
-                f"You already specify the feature names of {ntype} " \
-                f"as {fname_dict[ntype]}"
-        assert isinstance(feat_info[1], str), \
-            f"Feature name of {ntype} should be a string not {feat_info[1]}"
+        assert ntype not in fname_dict, (
+            f"You already specify the feature names of {ntype} "
+            f"as {fname_dict[ntype]}"
+        )
+        assert isinstance(
+            feat_info[1], str
+        ), f"Feature name of {ntype} should be a string not {feat_info[1]}"
         # multiple features separated by ','
         fname_dict[ntype] = feat_info[1].split(",")
 
     return fname_dict
 
 
-def get_edge_feat_info(folder, edge_feat_names):
-    # per edge type feature
+def get_edge_feat_info(edge_feat_names):
+    """Process edge feature names"""
     fname_dict = {}
     for feat_name in edge_feat_names:
         feat_info = feat_name.split(":")
-        assert len(feat_info) == 2, \
-                f"Unknown format of the feature name: {feat_name}, " + \
-                "must be EDGE_TYPE:FEAT_NAME"
-        etype = feat_info[0].split(',')
-        assert len(etype) == 3, \
-                f"EDGE_TYPE should have 3 strings: {etype}, " + \
-                "must be NODE_TYPE,EDGE_TYPE,NODE_TYPE:"
+        assert len(feat_info) == 2, (
+            f"Unknown format of the feature name: {feat_name}, "
+            + "must be EDGE_TYPE:FEAT_NAME"
+        )
+        etype = feat_info[0].split(",")
+        assert len(etype) == 3, (
+            f"EDGE_TYPE should have 3 strings: {etype}, "
+            + "must be NODE_TYPE,EDGE_TYPE,NODE_TYPE:"
+        )
         etype = ":".join(etype)
-        assert etype not in fname_dict, \
-                f"You already specify the feature names of {etype} " \
-                f"as {fname_dict[etype]}"
-        assert isinstance(feat_info[1], str), \
-            f"Feature name of {etype} should be a string not {feat_info[1]}"
+        assert etype not in fname_dict, (
+            f"You already specify the feature names of {etype} "
+            f"as {fname_dict[etype]}"
+        )
+        assert isinstance(
```

Comment:
You get feat_info from `feat_info = feat_name.split(":")`, it should already be a string. There is no need for this check.